### PR TITLE
Add OCI Object Storage Profiles

### PR DESCRIPTION
**Pull request by itemir**
This PR adds connection profiles for Oracle OCI Object Storage

### CODE REVIEW

File: profiles/OCI Object Storage us-ashburn-1.cyberduckprofile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Protocol</key>
+	<string>s3</string>
+	<key>Vendor</key>
+	<string>oracle-us-phoenix-1</string>
```
Comment (dkocher) [inline profiles/OCI Object Storage us-ashburn-1.cyberduckprofile]: Vendor strings should be unique among all profiles.

File: profiles/OCI Object Storage eu-frankfurt-1.cyberduckprofile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	</string>
+	<key>Region</key>
+	<string>eu-frankfurt-1</string>
+	<key>Regions</key>
+	<array>
+		<string>us-phoenix-1</string>
```
Comment (dkocher) [inline profiles/OCI Object Storage eu-frankfurt-1.cyberduckprofile]: As we are connecting to a non-global region endpoint it is probably not advisable to allow creating buckets for different regions – I make the assumption that these region specific requests for new buckets with a region different than the connected endpoint will fail?

**Comment by dkocher** [comment]
Would it be possible that you embed an icon that is based on our [template](https://github.com/iterate-ch/cyberduck/blob/master/profiles/assets/Template.psd)?

File: profiles/OCI Object Storage us-ashburn-1.cyberduckprofile (hunk 1 above)
Comment (itemir) [reply profiles/OCI Object Storage us-ashburn-1.cyberduckprofile]: This is a remnant from my previous testing. Do we need Regions key at all? When it is omitted, does Cyberduck pick the region from Region key while creating top level folders (i.e. buckets)?

File: profiles/OCI Object Storage eu-frankfurt-1.cyberduckprofile (hunk 2 above)
Comment (itemir) [reply profiles/OCI Object Storage eu-frankfurt-1.cyberduckprofile]: This is a remnant from my previous testing. Do we need Regions key at all? When it is omitted, does Cyberduck pick the region from Region key while creating top level folders (i.e. buckets)?

**Comment by itemir** [comment]
> Would it be possible that you embed an icon that is based on our template?

I am afraid it is beyond my image editing capabilities...

**Comment by itemir** [comment]
> Would it be possible that you embed an icon that is based on our template?

I managed to do it but didn't look good when resized. So removed it altogether.

**Comment by itemir** [comment]
Also removed the Regions key, looks like it is not required. 

Let me know how how the final version looks.

**Comment by itemir** [comment]
@dkocher I added an icon based on the Cyberduck template and added the regions back. Please have a look and let me know if this looks good.

**Pull request merged**